src/Contacts.ts

```diff
@@ -260,6 +260,15 @@ export class Contacts {
     timestampMs: number,
     peerAddress: string
   ): Promise<boolean> {
```

Comment:
is there a specific reason this method is async? looks like it could be sync

Reply:
Nope! good catch!